src/Appwrite/Platform/Task.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Appwrite\Platform;
+
+use Utopia\Database\Database;
+use Utopia\Platform\Action;
+use Exception;
+use InfluxDB\Database as InfluxDatabase;
+use Utopia\App;
+use Utopia\Cache\Adapter\Redis as RedisCache;
+use Utopia\Cache\Cache;
+use Utopia\CLI\Console;
+use Utopia\Database\Adapter\MariaDB;
+use Utopia\Registry\Registry;
+use Utopia\Logger\Log;
+use Throwable;
+
+abstract class Task extends Action
+{
+    protected function logError(Registry $register, Throwable $error, string $namespace, string $action)
+    {
+        $logger = $register->get('logger');
```

Comment:
This logger, can also be initialized as a DI resource.

Reply:
done